symfony/src/Service/SessionService.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\Service;
+
+use Pmi\Entities\Session;
+use Symfony\Component\DependencyInjection\ParameterBag\ParameterBagInterface;
+
+class SessionService
+{
+    private $params;
+
+    public function __construct(ParameterBagInterface $params)
+    {
+        $this->params = $params;
+    }
+
+    public function deleteKeys()
+    {
+        $sessionTimeout = 30 * 60;
```

Comment:
Can we get this from the environment service since it can change based on environment?